### PR TITLE
Creating a read me

### PR DESCRIPTION
**Pull request by costea32**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,55 @@
...
+```java
+
+public class User{
+
+  @IgnoreComparison
+  private string id;
+  
+  private string firstName;
+  
+  private string lastName;
```
Comment (cat-bug) [inline README.md]: did you mean `String`? 
also what about to provide parametrized comparison for one on String fields? like
`@StringComparison(caseSensitive = false)`
just for example of parametrized comparison

Comment (costea32) [reply README.md]: yep, Should be String, more about parameterized comparison will be added in wiki

File: README.md [hunk 2]
```diff
@@ -1,3 +1,55 @@
 # jEntityTest
 [![CircleCI](https://circleci.com/gh/TestMonkeys/jEntityTest/tree/develop.svg?style=svg)](https://circleci.com/gh/TestMonkeys/jEntityTest)
 [![codecov](https://codecov.io/gh/TestMonkeys/jEntityTest/branch/master/graph/badge.svg)](https://codecov.io/gh/TestMonkeys/jEntityTest)
+
+jEntityTest is a library that can compare two instances of a class based on their declared fields. Making it easier to check if they are equal. It also contains a hamcrest matcher so that you can use the comparison in assertions. The comparison can further be customized via annotations for each field.
```
Comment (monofrei) [inline README.md]: compares

File: README.md [hunk 3]
```diff
@@ -1,3 +1,55 @@
...
+  
+  private String firstName;
+  
+  private String lastName;
+  ...
+
+```
+There are a few more comparison strategies that can be applied for any field, that will impact how the comparison is done.
+
+# Comparison Strategies available
```
Comment (monofrei) [inline README.md]: add an example for each strategie

Comment (costea32) [reply README.md]: Adding more info in the wiki.

**Pull request merged**